### ISSUE TITLE
Error on submitting new a new mixin

### ISSUE THREAD

**Issue by escopecz** [mautic/website#87]
Go to http://staging.mautic.org/marketplace and hit the **Submit your Mautic Mixins** button while logged off.

```
Trying to get property of non-object
/var/www/html/themes/oc-theme-mautic-org/pages/mauticians/profile-marketplace_addedit.htm line 24
```

**Comment by kuzmany** [comment]
Thanks, I will fixed it.

**Comment by RCheesley** [comment]
I also had issues with addedit on the profile page which I was looking
into, so might be related!

**Comment by kuzmany** [comment]
@RCheesley add edit what? You are logged in/out? Please describe same way like @escopecz . I will fixed it then.

**Comment by RCheesley** [comment]
When logged in, with an article on the blog submitted, I found there was
some wrong logic and the edit button always showed view, rather than edit.

When I fixed that late last night, the edit feature did not work - it just
reloaded the page.

I was looking into it and I recall getting to find addedit as what should
be happening but I didn't get any further, so might be worth checking the
edit blog feature as well as the add/edit mixins.
